test/outtest4.F90

```diff
@@ -131,6 +132,10 @@ program outtest4
 
   call writsa ( -13, mxbfmg, mgbf, lmgbf )
 
+  ! Get Section 1 date (returns 8-byte ints for KIND_8).
+  call igetdate(mgbf, mear, mmon, mday, mour)
+  if (mear.ne.20 .or. mmon.ne.10 .or. mday.ne.1 .or. mour.ne.11) stop 6
```

Comment:
Nice! I like the way you checked all values, not just some of them.

Reply:
Thanks Alex, but just FYI that igetdate is a function, not a subroutine.  So it has a return value that could also be checked, but at a minimum it should at least be called as a function (not as a subroutine), and it should also be explicitly declared as "integer*4 igetdate" at the top of the routine, alongside of isetprm, ireadsb, igetmxby, and icbfms.